### PR TITLE
Fix function key definitions in terminfo/termcap

### PR DESCRIPTION
**Pull request by shimt**
Fix modifier key + F3 does not work in current terminfo/termcap.

Before (0.27.0):

| Key            | Kitty Output | Terminfo Variable | Terminfo Value |
| -------------- | ------------ | ----------------- | -------------- |
| F3             | SS3 OR       | kf3               | SS3 OR         |
| Shift F3       | CSI 13;2~    | kf15              | CSI 1;2R       |
| Ctrl F3        | CSI 13;5~    | kf27              | CSI 1;5R       |
| Alt F3         | CSI 13;3~    | kf51              | CSI 1;3R       |
| Shift + Alt F3 | CSI 13;4~    | kf63              | CSI 1;4R       |

After:

| Key            | Kitty Output | Terminfo Variable | Terminfo Value |
| -------------- | ------------ | ----------------- | -------------- |
| F3             | SS3 OR       | kf3               | SS3 OR         |
| Shift F3       | CSI 13;2~    | kf15              | **CSI 13;2~**       |
| Ctrl F3        | CSI 13;5~    | kf27              | **CSI 13;5~**    |
| Alt F3         | CSI 13;3~    | kf51              | **CSI 13;3~**       |
| Shift + Alt F3 | CSI 13;4~    | kf63              | **CSI 13;4~**     |

### CODE REVIEW

**Comment by mark2185** [comment]
I'm not sure where I should be asking this, but since `v0.27.0` I've noticed that `Ctrl F3` seems to not work in `tmux`, it just prints out `3;5~`, while `Ctrl F1`, `Ctrl F2` and `Ctrl F4` do work.

I'm using `v0.26.5` in the meantime.

What more info can I provide?

**Comment by kovidgoyal** [comment]
There isnt anything to provide, assuming you are running kitty 0.27.1 in
which this bug was fixed, it should work in tmux out of the box, it does
for me.

**Comment by mark2185** [comment]
_Huh, that's odd, it does work._

I must've had `v0.27.0` still running after updating. Sorry for the noise!

**Comment by mark2185** [comment]
Sorta kinda related, but now after updating to latest `kitty` I've discovered something yet again related to `tmux`.

Opening up `FZF` in `vim` (which brings up a popup terminal where the user can input text) allows the user to press `ctrl+/` to hide the preview of searched files, and that works, _but only in `tmux`_. 
When the combination `ctrl+/` is pressed _outside_ of tmux, it inserts `;5u`.

So, in short, running `vim +FZF` and pressing `ctrl+/` inputs `;5u`, but running `tmux && vim +FZF` and pressing `ctrl+/` works as expected.

`kitty --version` gives `0.28.1`. What else can I provide?

**Comment by kovidgoyal** [comment]
This will be a bug in vim, tmux filters out the kitty keyboard protocol
events. Update vim, or if it is already updated report the issue there.
And dont forget to follow the instructions in the kitty FAQ about vim.
